### ISSUE TITLE
DocFX - Material - Search won't work

### ISSUE THREAD

**Issue by GFMsoft** [ovasquez/docfx-material#16]
Hello Everyone,

currently i'm  working on generating a documentation for a project.
I want to use material as a template because the template fits all my needs.

Sadly, the searchbar won't work even with `"_enableSearch": "true"`  and  `"postProcessors": ["ExtractSearchIndex"] `.  :(
I followed the installation instructions, but no success.
I'm guessing it's a technical problem?

For debugging i'll post my docfx.json.
(src entry is a placeholder)

Looking forward to using your brilliant template with a working searchbar soon.
Thank you.

Greetings
Marx
____________

{
  "metadata": [
    {
      "src": [
        {
          "files": [
            "**.csproj"
          ],
          "src": "c:\\test\\testproject"
        }
      ],
      "dest": "api",
      "disableGitFeatures": false,
      "disableDefaultFilter": false
    }
  ],
  "build": {
    "content": [
      {
        "files": [
          "api/**.yml",
          "api/index.md"
        ]
      },
      {
        "files": [
          "articles/**.md",
          "articles/**/toc.yml",
          "toc.yml",
          "*.md"
        ]
      }
    ],
    "resource": [
      {
        "files": [
          "images/**"
        ]
      }
    ],
    "overwrite": [
      {
        "files": [
          "apidoc/**.md"
        ],
        "exclude": [
          "obj/**",
          "_site/**"
        ]
      }
    ],
    "dest": "_site",
	"globalMetadata": {
		"_enableSearch": "true",
		"_appTitle": "123",
		"_noindex": "false",
		"_noFollow": "false"
	},
    "globalMetadataFiles": [],
    "fileMetadataFiles": [],
    "template": [
        "default",
		"modern",
		"templates/material"
    ],
    "postProcessors": ["ExtractSearchIndex"],
    "markdownEngineName": "markdig",
    "noLangKeyword": false,
    "keepFileLink": false,
    "cleanupCacheHistory": false,
    "disableGitFeatures": false
  }
}

**Comment by ovasquez** [comment]
Hi @GFMsoft, I just tried building the DocFX-Material website using the latest version of DoxFX 2.70 and the search is working as expected.

The search feature is part of the default template, and I simply consolidated the documentation for what I consider what could be a very frequently used feature.

My suggestion is to try to enable the search using the default template and then switch back to the material template, or try with a new from scratch DocFX site and see if the search is working and take it from there.

Looking at your docfx.json, the only thing that stands out to me is the following, although I'm not sure if this is the problem:
```json
"files": [
"articles/.md",
"articles//toc.yml",  # Double slash in the toc.yml
"toc.yml",
"*.md"
]
```
